Touch Bar Simulator/AppDelegate.swift

```diff
@@ -17,6 +19,28 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
 		$0.button!.toolTip = "Right-click or option-click for menu"
 	}
 
+	struct KeyboardShortcutRecorder: SwiftUI.View {
+		var title: String
+		var shortcut: KeyboardShortcuts.Name
+		var body: some View {
+			HStack {
+				Text(title)
+				KeyboardShortcuts.Recorder(for: shortcut)
+			}
+		}
+	}
+
+	lazy var keyboardShortcutsVC = NSHostingController(rootView:
+		AnyView(
+			VStack(alignment: .center) {
+				Text("Keyboard Shortcuts:")
```

Comment:
I don't think this is needed. It's already clear to the user what this is as they clicked "Keyboard Shortcuts…".